### DISCUSSION TITLE
How to Prevent Highlighting Child Tag Inside Mj-text

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5322] [General]
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/52c918af-9554-4955-a804-a383d6327afd)

You may see that the outer component is already selected but when point the mouse over to the text it just highlighting it. but the grapesjsMJML package should prevent this what ever inside the mj-text should not be highlighted this is the correct behaviour

**Comment by artf** [comment]
Try this [option](https://github.com/GrapesJS/grapesjs/blob/76afd0dc56a4a28fe59aa7a0b1ab0c046a34f938/src/dom_components/config/config.ts#L28)

**Reply by RakulAgn** [reply]
I think this is not working
